alternate/Tenkuu Danzai Skelter Heaven (alternate).ass

```diff
@@ -26,213 +28,220 @@ Style: OPTL,Monotype Corsiva,22,&H00F8F091,&H000000FF,&H00000000,&H7F000000,-1,0
+Dialogue: 0,0:11:50.49,0:11:54.16,Default,,0,0,0,,However, her actual ability is unknown.
+Dialogue: 0,0:11:54.83,0:11:58.46,Default,,0,0,0,,And so, there will need to be lots of watching.
+Dialogue: 0,0:11:58.46,0:11:59.90,Default,,0,0,0,,It is sleeping.
+Dialogue: 0,0:12:00.60,0:12:05.90,Default,,0,0,0,,But if in the future it bared its fangs...{lit bare fangs, which is of course an idiom in English, but I dunno if it sounds right for this character}
+Dialogue: 0,0:12:12.88,0:12:17.02,Default,,0,0,0,,Commanding Officer, how long can I live?
+Dialogue: 0,0:12:17.62,0:12:19.85,Default,,0,0,0,,Are you scared of dieing?
+Dialogue: 0,0:12:22.22,0:12:23.46,Default,,0,0,0,,No.
+Dialogue: 0,0:12:23.46,0:12:27.06,Default,,0,0,0,,Um... We're created existences.
+Dialogue: 0,0:12:27.06,0:12:29.19,Default,,0,0,0,,My life span and health is wrong.
+Dialogue: 0,0:12:29.19,0:12:30.40,Default,,0,0,0,,I...
+Dialogue: 0,0:12:31.10,0:12:34.83,Default,,0,0,0,,I want to know my life.
+Dialogue: 0,0:12:35.30,0:12:39.07,Default,,0,0,0,,The company is responsible for your bodies.
+Dialogue: 0,0:12:39.07,0:12:40.54,Default,,0,0,0,,There's nothing to worry about.
+Dialogue: 0,0:12:41.87,0:12:45.48,Default,,0,0,0,,Just, there remains to be further research to be done on your lifespan.
+Dialogue: 0,0:12:46.78,0:12:50.52,Default,TLC,0,0,0,,Since you're the first generation, you are the research data.{Is that some sort of term there? 第一セライ or something?}{no idae}
+Dialogue: 0,0:12:51.52,0:12:53.25,Default,,0,0,0,,The pointless talking is finished.
+Dialogue: 0,0:12:53.25,0:12:55.45,Default,,0,0,0,,It's okay for you to follow orders.
+Dialogue: 0,0:12:55.45,0:12:56.89,Default,,0,0,0,,Good?
+Dialogue: 0,0:12:56.89,0:12:58.66,Default,,0,0,0,,I understand.
+Dialogue: 0,0:12:59.76,0:13:01.06,Default,,0,0,0,,That's good.
+Dialogue: 0,0:13:01.06,0:13:03.43,Default,,0,0,0,,Taichou's order is absolute.
+Dialogue: 0,0:13:03.43,0:13:06.43,Default,,0,0,0,,You really are always honest, huh?
+Dialogue: 0,0:13:07.03,0:13:07.70,Default,,0,0,0,,Eh?
+Dialogue: 0,0:13:19.41,0:13:21.75,Default,,0,0,0,,Rin, get a hold of yourself!
+Dialogue: 0,0:13:25.42,0:13:27.12,Default,,0,0,0,,I'm okay, I'm okay.
+Dialogue: 0,0:13:27.12,0:13:28.12,Default,,0,0,0,,I can still go!
+Dialogue: 0,0:13:28.45,0:13:30.49,Default,,0,0,0,,Rin... no.
+Dialogue: 0,0:13:30.49,0:13:31.76,Default,,0,0,0,,All units, retreat.
+Dialogue: 0,0:13:33.12,0:13:35.23,Default,TLC,0,0,0,,We will finish it off.
+Dialogue: 0,0:13:35.23,0:13:38.80,Default,,0,0,0,,When its weakness is known, we'll attack this guy and destroy it!
+Dialogue: 0,0:13:38.80,0:13:40.73,Default,,0,0,0,,Funagai! Are you crazy?
+Dialogue: 0,0:13:42.23,0:13:43.90,Default,,0,0,0,,This was just work.
+Dialogue: 0,0:13:45.67,0:13:48.71,Default,,0,0,0,,Thinking that, I somehow did it.
+Dialogue: 0,0:13:49.47,0:13:50.28,Default,,0,0,0,,But...
+Dialogue: 0,0:13:51.54,0:13:52.68,Default,,0,0,0,,They...
+Dialogue: 0,0:13:53.31,0:13:54.41,Default,,0,0,0,,were laughing.
```

Comment:
These got changed during timing to have leading ellipses. May as well add them here to make the merge easier.
```suggestion
Dialogue: 0,0:13:49.47,0:13:50.28,Default,,0,0,0,,But...
Dialogue: 0,0:13:51.54,0:13:52.68,Default,,0,0,0,,... They...
Dialogue: 0,0:13:53.31,0:13:54.41,Default,,0,0,0,,... were laughing.
```